### PR TITLE
[Doc] add explanations for REST API (function/source/sink)

### PR DESCRIPTION
**Pull request by Anonymitaet**
Fix https://github.com/apache/pulsar/issues/13542. Many thanks for your help!! @urfreespace 

Preview looks good:

Function:
image

Source:
image

Sink:
image

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@tuteng can you help review? Thanks

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/SourcesBase.java [hunk 1]
```diff
@@ -125,7 +129,21 @@ public void registerSource(
                                             + "  \"archive\": /connectors/pulsar-io-mysql-0.0.1.nar\n"
                                             + "  \"schemaType\": avro\n"
                                             + "}\n"
-                            )
+                                            + "Example 2 \n{\n"
```
Comment (tuteng) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/SourcesBase.java:132]: Maybe this example is not correct here, we may need a different example

**Review by tuteng — APPROVED** [review]
LGTM, Thanks

**Pull request merged**